### PR TITLE
[Released] v1.00.27-20210527

### PR DESCRIPTION
**Pull request by paxusactionbot**
This PR is generated by automation

### CODE REVIEW

File: lib-ui-constant/src/main/java/com/pax/us/pay/ui/constant/entry/InformationEntry.java [hunk 1]
```diff
@@ -25,6 +25,12 @@
...
     public static final String ACTION_DISPLAY_APPROVE_MESSAGE = "com.pax.us.pay.action.DISPLAY_APPROVE_MESSAGE";
 
-
+    /**
+     * The intent action of display transaction information <br>
+     * input: {@link EntryExtraData#PARAM_PACKAGE} <br>
+     * input: {@link EntryExtraData#PARAM_MESSAGE} <br>
+     * request: None <br>
+     */
+    public static final String ACTION_DISPLAY_TRANS_END = "com.pax.us.pay.action.DISPLAY_TRANS_END";//added for visa installment
```
Comment (Kim-Linhb) [inline lib-ui-constant/src/main/java/com/pax/us/pay/ui/constant/entry/InformationEntry.java:34]: 1. Is it a page action or a status?
2. Is the action name correct, display trans end, transaction information? does it relate to visa installment?

**Review by Kim-Linhb — CHANGES_REQUESTED** [review]
v1.00.27-20210527 was reviewed on 20210528.
Please see my comments.

File: lib-ui-constant/src/main/java/com/pax/us/pay/ui/constant/entry/InformationEntry.java (hunk 1 above)
Comment (MichealUfl) [reply lib-ui-constant/src/main/java/com/pax/us/pay/ui/constant/entry/InformationEntry.java:34]: This is a page action. Visa wants us to display a certain page when a visa installment transaction is approved. I think the name is not aacurate enough, I will modify them during the next release which will happen in this week.

Comment (Kim-Linhb) [reply lib-ui-constant/src/main/java/com/pax/us/pay/ui/constant/entry/InformationEntry.java:34]: Got it.
And please modify the descriptions as well.

Comment (Kim-Linhb) [reply lib-ui-constant/src/main/java/com/pax/us/pay/ui/constant/entry/InformationEntry.java:34]: @MichealUfl I didn't find update on any commits, Please modify it ASAP.

Comment (Kim-Linhb) [reply lib-ui-constant/src/main/java/com/pax/us/pay/ui/constant/entry/InformationEntry.java:34]: See df474c0

**Comment by Kim-Linhb** [comment]
Closed cuz #41

**Pull request closed without merging**